cmd/dep/testdata/init/case2/thing.input.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2016 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"github.com/golang/notexist/foo/bar"
+	"github.com/sdboyer/deptestdos"
+)
+
+func main() {
+	err := nil
+	if err != nil {
+		deptestdos.diMeLo(bar.Qux)
```

Comment:
While we don't currently do any actual code-level analysis (beyond import statements) during solving, it's possible we might add it in the future. If/when we do, these tests may start erroneously failing if they refer to nonexistent members in the deptest packages.

To guard against that possibility, let's make sure these input fixtures are valid uses of those packages - make sure you reference them somehow (so that it's not an unused import), but also make sure to only use types that are actually present.

Boring, I know 😈